spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -335,10 +335,14 @@ public DeltaWriter<InternalRow> createWriter(int partitionId, long taskId) {
       OutputFileFactory dataFileFactory =
           OutputFileFactory.builderFor(table, partitionId, taskId)
               .format(context.dataFileFormat())
+              .operationId(context.queryId())
+              .suffix("data")
```

Comment:
Can we leave the suffix out for data files? That way we don't make names any longer and don't change the format for most of the names in the table.

Reply:
sure, I've removed all `.suffix("data")` from the code so that we only have `.suffix("deletes")`